hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -668,29 +670,63 @@ public void revokeS3Secret(String kerberosID) throws IOException {
    * {@inheritDoc}
    */
   @Override
-  public void createTenant(String tenantName) throws IOException {
-    Preconditions.checkArgument(Strings.isNotBlank(tenantName),
+  public void createTenant(String tenantId) throws IOException {
+    Preconditions.checkArgument(Strings.isNotBlank(tenantId),
         "tenantName cannot be null or empty.");
-    ozoneManagerClient.createTenant(tenantName);
+    ozoneManagerClient.createTenant(tenantId);
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public void createTenant(String tenantId, OmTenantArgs omTenantArgs)
```

Comment:
Good point.

Reply:
Neither of them are protobuf classes, but the convention for bucket and volume creation is to not use the same class for client configuration and OM representation. I think we should be following the same pattern as existing requests like bucket and volume create here.

Reply:
Will add `TenantArgs`. Though I am not entirely sure why the client wouldn't just use `OmVolumeArgs` for `createVolume`. It might have something to do with the extra metadata (ObjectID/UpdateID).

Reply:
Done.